doc/classes/AnimatedTexture.xml

```diff
@@ -59,7 +59,6 @@
 		<member name="pause" type="bool" setter="set_pause" getter="get_pause" default="false">
 			If [code]true[/code], the animation will pause where it currently is (i.e. at [member current_frame]). The animation will continue from where it was paused when changing this property to [code]false[/code].
 		</member>
-		<member name="resource_local_to_scene" type="bool" setter="set_local_to_scene" getter="is_local_to_scene" overrides="Resource" default="false" />
```

Comment:
I don't know why doctool insists on removing these, it doesn't seem related to this PR but a left over from the PR that broke the texture classes up into separate files (seems to be a recent change)?

Reply:
Have you tried rebasing?

Reply:
yeah on master from a few hours ago :)

Reply:
As discussed on RC, this appears to happen as a result of defining the RD texture types in the register_server_classes code. Moving the registrations to scene makes this issue go away:
```
diff --git a/scene/register_scene_types.cpp b/scene/register_scene_types.cpp
index 1c2ed16e63..d86ae2ed28 100644
--- a/scene/register_scene_types.cpp
+++ b/scene/register_scene_types.cpp
@@ -226,6 +226,7 @@
 #include "scene/theme/theme_db.h"
 
 #include "scene/main/shader_globals_override.h"
+#include "servers/rendering/renderer_rd/scene/texture_rd.h"
 
 #ifndef _3D_DISABLED
 #include "scene/3d/area_3d.h"
@@ -887,6 +888,13 @@ void register_scene_types() {
        GDREGISTER_CLASS(PlaceholderCubemap);
        GDREGISTER_CLASS(PlaceholderCubemapArray);
 
+       GDREGISTER_CLASS(Texture2DRD);
+       GDREGISTER_ABSTRACT_CLASS(TextureLayeredRD);
+       GDREGISTER_CLASS(Texture2DArrayRD);
+       GDREGISTER_CLASS(TextureCubemapRD);
+       GDREGISTER_CLASS(TextureCubemapArrayRD);
+       GDREGISTER_CLASS(Texture3DRD);
+
        GDREGISTER_CLASS(Animation);
        GDREGISTER_CLASS(AnimationLibrary);
 
diff --git a/servers/register_server_types.cpp b/servers/register_server_types.cpp
index 01bb0bab7c..7cc9a82699 100644
--- a/servers/register_server_types.cpp
+++ b/servers/register_server_types.cpp
@@ -69,7 +69,6 @@
 #include "physics_server_3d.h"
 #include "physics_server_3d_wrap_mt.h"
 #include "rendering/renderer_compositor.h"
-#include "rendering/renderer_rd/scene/texture_rd.h"
 #include "rendering/rendering_device.h"
 #include "rendering/rendering_device_binds.h"
 #include "rendering_server.h"
@@ -248,13 +247,6 @@ void register_server_types() {
        GDREGISTER_CLASS(RDShaderFile);
        GDREGISTER_CLASS(RDPipelineSpecializationConstant);
 
-       GDREGISTER_CLASS(Texture2DRD);
-       GDREGISTER_ABSTRACT_CLASS(TextureLayeredRD);
-       GDREGISTER_CLASS(Texture2DArrayRD);
-       GDREGISTER_CLASS(TextureCubemapRD);
-       GDREGISTER_CLASS(TextureCubemapArrayRD);
-       GDREGISTER_CLASS(Texture3DRD);
-
        GDREGISTER_CLASS(CameraFeed);
```

Reply:
We did end up deciding to move the classes into `scene` as well. But once we decide to move the renderer to a module we should add this back in.